Makefile

```diff
@@ -96,12 +96,6 @@ format: $(GOIMPORTS)
 # ------------------------------------------------------------------------------
 #  dependencies
```

Comment:
While we're making the switch to go modules, we should begin tracking dependencies. This guarantees a reproducible environment for future builds, as well as relieving the strain of manual dependency management for developers.

Documentation on doing so is here: https://github.com/golang/go/wiki/Modules#how-can-i-track-tool-dependencies-for-a-module